src/adapters/OrganizationsJSONAdapter.test.js

```diff
@@ -0,0 +1,54 @@
+import {isObservable} from 'rxjs';
+
+import organizations from '../data/organizations';
+import OrganizationsJSONAdapter from './OrganizationsJSONAdapter';
+
+describe('Organization JSON Adapter', () => {
+  let ID;
+  let organizationsJSONAdapter;
+
+  beforeEach(() => {
+    [ID] = Object.keys(organizations);
+    organizationsJSONAdapter = new OrganizationsJSONAdapter(organizations);
+  });
+
+  afterEach(() => {
+    organizationsJSONAdapter = null;
+  });
+
+  describe('getOrg()', () => {
+    test('returns an observable', () => {
+      expect(isObservable(organizationsJSONAdapter.getOrg())).toBeTruthy();
+    });
+
+    test('returns an organization data', (done) => {
+      organizationsJSONAdapter.getOrg(ID).subscribe((data) => {
```

Comment:
please rename `data` to the more specific `organization` (or `org`)

Reply:
done